chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -10,11 +10,12 @@ import chisel3.internal._
 import chisel3.internal.Builder.pushCommand
 import chisel3.internal.firrtl._
 import chisel3.internal.sourceinfo.{SourceInfo, DeprecatedSourceInfo, VecTransform, SourceInfoTransform}
+import chisel3.NotStrict.CompileOptions
```

Comment:
It is unclear upon first glance which functions needed this implicit in scope.

However, don't forget there is always an option to not choose a compiler mode and have the function that needs a compiler mode to take a CompileOption implicitly and then capture the options via implicit resolution (i.e. somewhat statically) at the user's callsite.